docs.phtml

```diff
@@ -31,25 +31,25 @@
   </ul>
 
-  <p class="justify"><?php echo T_("The <b>Help Manual</b> is designed to be a quick reference of how to accomplish specific tasks and how to use the features in <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is designed to be an in depth guide to the concepts behind using <span class=\"gnucash\">GnuCash</span> with a tutorial to show how to put those concepts into practice.");?></p>
+  <p class="justify"><?php echo T_("The <b>Help Manual</b> details specific tasks and how to use features <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is an in-depth guide to <span class=\"gnucash\">GnuCash</span> tutorial.");?></p>
 
   <p class="justify"><?php
     /* Todo: menu.phtml has a similar link. They should be unified with the other language stuff.*/
     /* Translators: If wiki pages in your laguage exists, consider to replace "De"/"Deutsch" by your language.*/
-    echo T_("Other resources are the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
+    echo T_("Other resources are to be found in the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
     ?></p>
 
   <p class="justify"><?php
-  echo T_("You can also send an email to the <a href=\"mailto:gnucash-user@gnucash.org\">gnucash-user</a> mailing list if you cannot find a satisfactory answer to your question within either the <b>Help Manual</b> or the <b>Tutorial and Concepts Guide</b>. We <b>want</b> feedback from you, it is also through your comments that we know how to improve the documentation.");
+  echo T_("Send an e-mail to the <a href=\"mailto:gnucash-user@gnucash.org\">gnucash-user</a> mailing list if you have tried the <b>Help Manual</b> and the <b>Tutorial and Concepts Guide</b>. Feedback and hints on what resources are missing is greatly appreciated.");
   /* Translators: %s is a link */
-  printf (T_("Additionally, you can talk to someone via IRC at %s about your question."),
+  printf (T_("Additionally, you can talk to someone by posting your question on IRC at %s."),
       sprintf ("<a href='%s'><i>irc.gnome.org</i> %s <b>#gnucash</b></a>", \URL_IRC,
           /* Translators: IRC channel */
           T_("channel")));
   ?></p>
 
   <div class="dialog note">
-    <p class="justify"><?php echo T_("<span class=\"gnucash\">GnuCash</span>'s documentation has been created by its community. See the <a href=\"develdocs.phtml\">Writing Documentation page</a> if you are interested in contributing to this effort.");?></p>
+    <p class="justify"><?php echo T_("The <span class=\"gnucash\">GnuCash</span> documentation is a community creation. The <a href=\"develdocs.phtml\">Writing Documentation page</a> helps you improve it.");?></p>
   </div>
```

Comment:
Prefer:

The GnuCash documentation is a community creation. The Writing Documentation page provides specific instruction and guidance on how you can contribute.

Reply:
Too long-winded. Keep it sniple snupple.

Reply:
@comradekingu It may be longer but it is also idiomatic and correct. Your version is neither. @sunfish62 is a professional editor. You would be well advised to learn from him.

Reply:
In any professional capacity, I don't understand why referencing "it" is somehow worse than leaving "contribute" up in the air? I'll try to tie it together.

Reply:
Maybe "contribute" is up in the air for you, but it isn't for me. The difference between your version and mine is three words. And, for the record, your version suggests that a user's writing contributions would improve the community effort, rather than the actual documentation.